src/libs/actions/Policy/DistanceRate.ts

```diff
@@ -128,10 +128,10 @@ function enablePolicyDistanceRates(policyID: string, enabled: boolean) {
 
     if (!enabled) {
         const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`];
-        const customUnitID = Object.keys(policy?.customUnits ?? {}).at(0) ?? '';
-        const customUnit = customUnitID ? policy?.customUnits?.[customUnitID] : undefined;
+        const distanceRateCustomUnit = getDistanceRateCustomUnit(policy);
```

Comment:
```suggestion
        const customUnit = getDistanceRateCustomUnit(policy);
```

NAB. In this context I think it's clear that we are referring to distance rate. Also this is more consistent with the other variable name (customUnitID)